tests/failpoint/db_failpoint_test.go

```diff
@@ -122,3 +123,34 @@ func TestFailpoint_ResizeFileFail(t *testing.T) {
 
 	require.NoError(t, err)
 }
+
+func TestFailpoint_LackOfDiskSpace(t *testing.T) {
+	db := btesting.MustCreateDB(t)
+
+	err := gofail.Enable("lackOfDiskSpace", `return("grow somehow failed")`)
+	require.NoError(t, err)
+
+	tx, err := db.Begin(true)
+	require.NoError(t, err)
+
+	err = tx.Commit()
+	require.Error(t, err)
+	require.ErrorContains(t, err, "grow somehow failed")
+
+	err = tx.Rollback()
+	require.NoError(t, err)
```

Comment:
I believe it also needs to be updated after you fixed above comment.

Reply:
Makes sense. Fixed!